mautic/mautic
# Landing Pages Tests

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? | N/A
| Deprecations? | N/A

#### Description:
This PR creates a set of unit tests for the PageBundle. 

#### Steps to test this PR:
1. Apply this PR
2. Run `composer test`. All tests should pass.
3. Create or Edit a Landing Page, everything should work as before.
